sass/footer/_footer.scss

```diff
@@ -0,0 +1,51 @@
+footer {
+  height: 50%;
+  padding: 20px;
+  background-color: #121212;
+  font-family: 'Raleway', sans-serif;
+
+  .map-wrapper {
+    float: left;
+    background-color: #ffffff;
+    width: 50%;
+    height: 100%;
+  }
+
+  .contact-wrapper {
+    vertical-align: middle;
+    left:50%;
+    height: 100%;
+
+    .contact-list{
+      list-style: none;
+      text-align: center;
+
+      .bargar-wrapper {
+        display: inline-block;
+        margin: 10px 40px 0 40px;
+        text-align: justify;
+
+        .bargar-list {
+          list-style: none;
+          font-size: 14px;
+          color: #ffffff;
+          font-weight: map-get($fontWeights, 'medium');
```

Comment:
I like that weights are in a map. You can also use simple mixin to save yourself some time…
```
@mixin fontWeight($weightName) {
  font-weight: map-get($fontWeights, $weightName);
}
```
And now instead of writing this: 
```
font-weight: map-get($fontWeights, 'medium');
```
simply write:
```
@include fontWeight('extraBold')
```